src/languages/EN-mx.json

```diff
@@ -0,0 +1,43 @@
+{
```

Comment:
why did we named the file name as EN-mx.json? Rest all file names are en or es

Reply:
will delete the extra files